Resources/translations/SonataDoctrinePHPCRAdmin.pl.xliff

```diff
@@ -33,7 +33,7 @@
             <trans-unit id="label_type_contains_words">
                 <source>label_type_contains_words</source>
                 <target>zawiera słowa</target>
-            </trans-unit>            
+            </trans-unit>
         </body>
-     </file>
-</xliff>
+    </file>
+</xliff>
```

Comment:
Please don't change line 39 and configure your editor to keep a newline at end of file.